README.md

```
@@ -39,41 +38,55 @@ git clone https://github.com/tschm/config-templates.git
+- Install `uv` and `uvx` into the `bin/` directory
+- Create a Python virtual environment in `.venv/`
+- Install all project dependencies from `pyproject.toml`
+
 Both the `.venv` and `bin` directories are listed in `.gitignore`.
 
 ## 📋 Available Tasks
 
-Run `./bin/task --list-all` to see all available tasks:
+Run `make help` to see all available targets:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)